easybuild/easyconfigs/l/libGridXC/libGridXC-1.1.0-gompi-2022a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'libGridXC'
+version = '1.1.0'
+
+homepage = 'https://gitlab.com/siesta-project/libraries/libgridxc'
+description = """A library to compute the exchange and correlation energy
+ and potential in spherical (i.e. atoms) or periodic systems."""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'opt': True}
+
+source_urls = ['https://gitlab.com/siesta-project/libraries/libgridxc/-/archive/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e7883e57a4db2438ee59437740291c06e0cfe1c8ae1901e4001f32312307e46a']
+
+configopts = "-DWITH_MPI=ON -DWITH_LIBXC=ON"
+
+dependencies = [('libxc', '5.2.3')]
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+]
+
+parallel = 1
```

Comment:
Not sure this is needed or not

Reply:
On my cluster it also succeeds the build without the `parallel = 1` (using 64 cores for compilation). But I don't know how robust this is and if it can fail under certain circumstances.

Reply:
Lets remove it. Some projects actually do set up dependency tracking for mod files properly, so lets not limit it without knowing it has ever failed

Reply:
Okay, I removed the `parallel = 1` again